td-agent/yum/td-agent.spec.in

```diff
@@ -86,6 +86,14 @@ rake build:rpm_config TD_AGENT_STAGING_PATH=%{buildroot}
 rake build:rpm_old_config TD_AGENT_STAGING_PATH=%{buildroot}
 %endif
 rake build:all TD_AGENT_STAGING_PATH=%{buildroot}
+mkdir -p %{buildroot}%{_mandir}/man1
+cp @PACKAGE@/debian/*.1 %{buildroot}%{_mandir}/man1/
+for man in `find %{buildroot} -type f -name '*.1'`; do
+    gzip ${man}
+done
+for man in `find %{buildroot} -type f -name '*.3'`; do
```

Comment:
Memo:

If it's not exist, man files in gems and other bundled softwares will cause following warnings:

```
td-agent.x86_64: W: manpage-not-compressed gz /opt/td-agent/share/man/man3/jemalloc.3
This manual page is not compressed with the gz compression method (does not
have the gz extension). If the compression does not happen automatically when
the package is rebuilt, make sure that you have the appropriate rpm helper
and/or config packages for your target distribution installed and try
rebuilding again; if it still does not happen automatically, you can compress
this file in the %install section of the spec file.
...
```
Probably td-agent users won't use such man files, so we can just remove such files.
But I'm not sure about it. I think compressing all man files is not so bad workaround.